Functionality.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+#include <string.h>
+#include "Functionality.h"
+
+void CreateInformat(int lowLimit,int highLimit,int counter, char* buff)
+{
+  
+char Data[100];
+sprintf(Data,"%s\n%d-%d, %d\n",CSVFormat,lowLimit,highLimit,counter);
```

Comment:
you can directly sprintf to the `buff` itself?

Reply:
for other test cases , i need to store and each time value is different so using as a temporary container.

Reply:
That is done now... i was thinking to clear it first before using for other test cases but its working fine now thanks